terratorch/tasks/regression_tasks.py

```diff
@@ -401,5 +421,5 @@ def model_forward(x):
         if self.tiled_inference_parameters:
             y_hat: Tensor = tiled_inference(model_forward, x, 1, self.tiled_inference_parameters)
         else:
-            y_hat: Tensor = self(x).output
+            y_hat: Tensor = self(x, **rest).output
```

Comment:
@Joao-L-S-Almeida can you please add a comment in the code what **rest is and why it makes sense? Will make maintenance easier

Reply:
Same structure as in the other steps. It's additional keys from the dataset (e.g. location or timestamp).